python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1299,3 +1299,82 @@ def writeConfig(self, butler, clobber=False, doBackup=True):
         """No config to write.
         """
         pass
+
+
+class MakeCcdVisitTableConnections(pipeBase.PipelineTaskConnections,
+                                   dimensions=("instrument",),
+                                   defaultTemplates={}):
+    visitSummaries = pipeBase.connectionTypes.Input(
+        doc="Per-visit consolidated exposure metadata from ConsolidateVisitSummaryTask",
+        name="visitSummary",
+        storageClass="ExposureCatalog",
+        dimensions=("instrument", "visit"),
+        multiple=True,
+        deferLoad=True,
+    )
+    outputCatalog = connectionTypes.Output(
+        doc="CCD and Visit metadata table",
+        name="CcdVisitTable",
+        storageClass="DataFrame",
+        dimensions=("instrument",)
+    )
+
+
+class MakeCcdVisitTableConfig(pipeBase.PipelineTaskConfig,
+                              pipelineConnections=MakeCcdVisitTableConnections):
+    pass
+
+
+class MakeCcdVisitTableTask(CmdLineTask, pipeBase.PipelineTask):
+    """Produce a `ccdVisitTable` from the `visitSummary` exposure catalogs.
+    """
+    _DefaultName = 'makeCcdVisitTable'
+    ConfigClass = MakeCcdVisitTableConfig
+
+    def run(self, visitSummaries):
+        """ Make a table of ccd information from the `visitSummary` catalogs
+        Parameters
+        ----------
+        visitSummaries : `list` of `lsst.afw.table.ExposureCatalog`
+            List of exposure catalogs with per-detector summary information.
+        Returns
+        -------
+        result : `lsst.pipe.Base.Struct`
+            Results struct with attribute:
+                - `outputCatalog`
+                    Catalog of ccd and visit information.
+        """
+        ccdEntries = []
+        for visitSummary in visitSummaries:
+            visitSummary = visitSummary.get()
+            visitInfo = visitSummary[0].getVisitInfo()
+
+            ccdEntry = {}
+            summaryTable = visitSummary.asAstropy()
+            selectColumns = ['id', 'visit', 'physical_filter', 'ra', 'decl', 'zenithDistance', 'zeroPoint',
+                             'skyBg', 'skyNoise', ]
+            ccdEntry = summaryTable[selectColumns].to_pandas().set_index('id')
+            ccdEntry.rename({"physical_filter": "filterName", "visitId": "visit"})
+            ccdEntry['ccdVisitId'] = [row.getVisitInfo().getExposureId() for row in visitSummary]
```

Comment:
This is not correct.  The `getExposureId()` call does not give what you want, and should not be used for anything (it's per visit but combines ccd info, but from a totally random ccd I think?).  See https://jira.lsstcorp.org/browse/RFC-459 which has never been implemented (sigh).  Right now I think we need to use the butler packer infrastructure for this.  But this is something that should be brought up on slack on what value we want here.  But it's is definitely not this.

Reply:
Ok, I compared `getExposureId()` with the gen2 method `dataRef.get('ccdExposureId')` (based on what is used here: https://github.com/lsst/pipe_tasks/blob/1e789892caf5a6dcf760974bf4037902b9f858dc/python/lsst/pipe/tasks/postprocess.py#L302) and got the same results for a few visits and ccds, but maybe I just got lucky. I'll bring it up on slack.

Reply:
If this is from the `visitInfo`, then there's just one number per visit.  Presumably we need something that is per detector here?

Reply:
For this, `getVisitInfo()` is called for each ccd in the visitSummary. Most of the attributes in `getVisitInfo` are the same for all of the rows, but ExposureId is different. For example, for visit 30500, you get 
```
ipdb> ccdEntry['ccdVisitId']
id
0      6100000
1      6100001
2      6100002
3      6100003
4      6100004
        ...   
99     6100099
100    6100100
101    6100101
102    6100102
103    6100103
Name: ccdVisitId, Length: 103, dtype: int64
```

Reply:
@TallJimbo helped with this question and a packer is used now.